### PR TITLE
Add support to build aarch64 wheels

### LINKED ISSUE #900: Missing aarch64 wheels
Problem:
On aarch64 architecture, `pip install Brotli` downloads the source distribution of it and build it. Directly downloading the wheel file and installing the same is quite efficient rather than building and installing from the source code. Also, it requires development environments to be installed on the user's machine. 

Overall, it is painful to install `Brotli` on aarch64 as it doesn't have aarch64 wheels on PyPi.

Resolution:
 `pip` should be able to download aarch64 wheels from PyPi and install same.

@eustas and team, please let me know your interest here. I can help you in building and uploading aarch64 wheels on PyPi.

### PR DESCRIPTION
**Pull request by odidev**
Closes https://github.com/google/brotli/issues/900

@anthrotype, can you please review this PR? Thanks.

### CODE REVIEW

**Comment by odidev** [comment]
[Travis-ci.org](http://www.travis-ci.org) will be shutting down soon so the repository needs to be migrated on [travisci.com](http://www.travis-ci.com). In this PR I have used arm64-graviton2 to build AArch64 artifacts which is not supported on [travis-ci.org](http://www.travis-ci.org).

Please let me know your thoughts on migrating to [travisci.com](http://www.travis-ci.com).

**Comment by odidev** [comment]
@anthrotype, Could you please review this PR?

File: .travis.yml [hunk 1]
```diff
@@ -20,53 +20,75 @@ services: docker
...
+        - MB_PYTHON_VERSION=3.6
+        - BUILD_SDIST=true
     - os: linux
       env:
-        - MB_PYTHON_VERSION=2.7
+        - MB_PYTHON_VERSION=3.6
         - PLAT=i686
-        - UNICODE_WIDTH=16
     - os: linux
+      arch: arm64-graviton2
```
Comment (anthrotype) [inline .travis.yml:32]: are you sure you need to select this on travis side? I would think it is enough to tell multibuild which docker image to use (in your case "xenial_arm64v8"), and it should not matter which host linux you run docker from. Have you tried without setting arch, dist, virt, group etc and only exporting those multibuild environment variables?

File: .travis.yml [hunk 2]
```diff
@@ -20,53 +20,75 @@ services: docker
 
 matrix:
   include:
-    - os: linux
-      env: MB_PYTHON_VERSION=2.7
```
Comment (anthrotype) [inline .travis.yml:24]: any reasons why you drop python2.7? It should not be related to your specific change

**Comment by anthrotype** [comment]
about the migration from travis.org to travis.com, I would rather migrate it to Github actions, but that's a different issue.

File: .travis.yml (hunk 1 above)
Comment (odidev) [reply .travis.yml:32]: @anthrotype travis provides two types of instances for arm64 i.e. arm64 and arm64-graviton2, to use arm64-graviton2 we need to specify these settings like arch, dist, virt etc..

File: .travis.yml (hunk 2 above)
Comment (odidev) [reply .travis.yml:24]: It is no more maintained by python community, so I removed it. Please let me know if it's required, I will reverse the changes.

**Comment by odidev** [comment]
> about the migration from travis.org to travis.com, I would rather migrate it to Github actions, but that's a different issue.

I tried using arm64 instance on travis-ci.org but its failing with error fatal: `Not a git repository: /home/travis/build/odidev/brotli-wheels/src/brotli-15868/.git/modules/research/esaxx`.
Please let me know if you have any plans to move to github actions. I can help with this.

**Comment by odidev** [comment]
@anthrotype, can you please let me know your interest on migrating to GHA?

**Comment by anthrotype** [comment]
@odidev I'm taking care of it right now, I'll make sure of building aarch64 linux wheels as well. Stay tuned

**Pull request closed without merging**